src/main/java/com/ericsson/eiffel/semantics/events/Batch.java

```diff
@@ -1,17 +1,17 @@
-/*
-    Copyright 2018 Ericsson AB.
-    For a full list of individual contributors, please see the commit history.
-    Licensed under the Apache License, Version 2.0 (the "License");
-    you may not use this file except in compliance with the License.
-    You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-    Unless required by applicable law or agreed to in writing, software
-    distributed under the License is distributed on an "AS IS" BASIS,
-    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-    See the License for the specific language governing permissions and
-    limitations under the License.
-*/
+/*******************************************************************************

```

Comment:
Why change the formatting on the copyright header? Isn't it easier to keep it as is with multi-line comment syntax?

Reply:
> Why change the formatting on the copyright header? Isn't it easier to keep it as is with multi-line comment syntax?

These classes are generated from the schema, 
The header is not generating by jsonschema2pojo library, so today we are placing manually the header.